app/views/static/ems_cloud/arbitration_profile_edit.html.haml

```diff
@@ -1,105 +1,110 @@
-    .form-group
-      %label.col-md-2.control-label{"for" => "cloud_subnet_id"}
-        = _('Cloud Subnet')
-      .col-md-8
-        %select{"ng-model"    => "arbitrationProfileModel.cloud_subnet_id",
-                "name"        => "cloud_subnet_id",
-                "checkchange" => ""}
-          %option{"value" => ""}
-            = "<#{_('Choose')}>"
-          %option{"ng-repeat"   => "cloud_subnet in arbitrationProfileModel.cloud_subnets track by $index",
-                  "value"       => "{{cloud_subnet.id}}"}
-            {{cloud_subnet.name}}
-    .form-group
-      %label.col-md-2.control-label{"for" => "security_group_id"}
-        = _('Security Group')
-      .col-md-8
-        %select{"ng-model"    => "arbitrationProfileModel.security_group_id",
-                "name"        => "security_group_id",
-                "checkchange" => ""}
-          %option{"value" => ""}
-            = "<#{_('Choose')}>"
-          %option{"ng-repeat"   => "security_group in arbitrationProfileModel.security_groups track by $index",
-                  "value"       => "{{security_group.id}}"}
-            {{security_group.name}}
-    :javascript
-      miqInitSelectPicker();
+  .form-group{"ng-class" => "{'has-error': angularForm.name.$invalid}"}
+    %label.col-md-2.control-label{"for" => "name"}
+      = _("Name")
+    .col-md-8
+      %input.form-control{"type"        => "text",
+                          "id"          => "name",
+                          "name"        => "name",
+                          "ng-model"    => "arbitrationProfileModel.name",
+                          "maxlength"   => 20,
+                          "required"    => "",
+                          "checkchange" => "",
+                          "auto-focus"  => ""}
+      %span.help-block{"ng-show" => "angularForm.name.$error.required"}
+        = _("Required")
+  .form-group{"ng-class" => "{'has-error': angularForm.description.$invalid}"}
+    %label.col-md-2.control-label{"for" => "description"}
+      = _("Description")
+    .col-md-8
+      %input.form-control{"type"        => "text",
+                          "id"          => "description",
+                          "name"        => "description",
+                          "ng-model"    => "arbitrationProfileModel.description",
+                          "maxlength"   => 50,
+                          "required"    => "",
+                          "checkchange" => ""}
+      %span.help-block{"ng-show" => "angularForm.description.$error.required"}
+        = _("Required")
+  .form-group
+    %label.col-md-2.control-label{"for" => "authentication_id"}
+      = _('Key Pair')
+    .col-md-8
+      %select{"ng-model"    => "arbitrationProfileModel.authentication_id",
+              "name"        => "authentication_id",
+              'ng-options'  => 'authentication.id as authentication.name for authentication in arbitrationProfileModel.authentications track by authentication.id',
```

Comment:
@himdel @AparnaKarve i had already tried this solution and didn't seem to work for me. i can push up changes from one of my stashes.